examples/nextjs-typescript/pages/properties/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { Heading, Wrap } from "@chakra-ui/core"
+import { Layout } from "../../components/Layout"
+import { properties } from "../../utils/sample-data"
+import { PropertySummary } from "../../components/PropertySummary"
+import { NextChakraLink } from "../../components/NextChakraLink"
+
+const PropertiesPage = () => (
+  <Layout title="Next.js + TypeScript example | View properties">
+    <Heading mb={4}>Available this weekend</Heading>
+    <Wrap>
+      {properties.map((property) => (
+        <NextChakraLink
```

Comment:
I think we should iterate on this (not in this PR, just sometime later) to make the summaries function as [semantic breakout buttons](https://hankchizljaw.com/wrote/create-a-semantic-breakout-button-to-make-an-entire-element-clickable/). This is totally fine for now though!

Reply:
Like that idea! 

TIL the word for those kind of buttons - Thanks